compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -652,18 +651,16 @@ scrapeInterfaceInstanceBinds ::
      Env
   -> [(Var, GHC.Expr CoreBndr)]
   -> MS.Map TypeConName InterfaceInstanceGroup
-scrapeInterfaceInstanceBinds env binds
-  | envLfVersion env `supports` featureSimpleInterfaces =
```

Comment:
Good catch! I hadn't realized that this envLfVersion check is premature and doesn't error out correctly.

Reply:
yeah, I think I added it 😅. At least I don't think we ever ran the risk of someone successfully writing an interface instance on LF 1.14 since importing packages from newer LF versions is forbidden. I do take the lesson that the scraping part shouldn't be too clever about what to scrape or not scrape.